firmware/development/kampela/Readme.md

```diff
@@ -0,0 +1,34 @@
+Wee are trying to build:
+
+cargo build --release
+
+... and get error that we don't have `thumbv8m.main-none-eabihf` target.
```

Comment:
This is better to place at the end in `# Troubleshooting` section, here a flawless flow should be shown.